### ISSUE TITLE
[Feature request] Unduly waving liquids

### ISSUE THREAD

**Issue by Inky-Inky** [Shanjaq/uhexen2#47]
The textures whose name start with "*" are displayed with an interesting distortion effect which is usually associated with liquids.
And sometimes it's also nice for magical portals too.
It has been used since Quake 1 teleporters that way.

Now, uhexen2 applies an additional effect to the brushes decorated with "*" textures.
Not only the 2D texture is distorted the traditional way, but also the 3D brush itself has its edges waving.

It makes sense for a lake, a river or that kind of things:

![hexen00](https://user-images.githubusercontent.com/72394209/98793993-d9f54400-2408-11eb-81d6-3fa89322f344.jpg)

But when it comes to magic fields/portals, it becomes a real nuisance.

The round magic field wrapping the dome here awkwardly reveals the stuco below because it can't stay still in its wrapping position:

![hexen02](https://user-images.githubusercontent.com/72394209/98794413-512ad800-2409-11eb-8dee-62dc1f3ced9c.jpg)

The carefully designed portal doesn't look carefully designed anymore because one can clearly see the day light between the magic field and the frame:

![hexen03](https://user-images.githubusercontent.com/72394209/98794620-8b947500-2409-11eb-8167-4eb87b9e4f4f.jpg)

My request would be to have a way to switch off the waving edges to keep only the vanilla 2D texture distortion.
A possible idea that would keep backward compatibility with already released maps' behavior: switch off the 3D waving effect when the "*" texture has also a special ending character (say "!" or whatever).
Example:
***lowlight** is the vanilla water texture which 2D warps + 3D waves
Its ***lowlight!** duplicate would 2D warp only

(and if ever the map would be run on another engine - which is highly unlikely because uhexen2-shanjaq is the best of all, no question - ***lowlight!** would behave just like ***lowlight** and cause nothing bad)

**Comment by Inky-Inky** [comment]
Aaarrrgh! Another example:

![image](https://user-images.githubusercontent.com/72394209/101157292-2c9dd680-362a-11eb-8349-73ffaee04c77.png)

"*Thanks*" to the waving, that cistern's sides become see-through. Ugly. :°°-0

**Comment by DelusionalBear** [comment]
yeah, I'm also would love this effect to go away or, at least, turned off by default
